packages/rspack/src/index.ts

```diff
@@ -7,6 +7,8 @@ export * from "./Stats";
 export * from "./MultiStats";
 export * from "./ChunkGroup";
 export * from "./NormalModuleFactory";
+export * from "./RuntimeGlobals";
```

Comment:
Please avoid using `export *`, using this may cause export a field by accident in the future. Please use `export { RuntimeGlobals } from "./xxx"`

Reply:
No problem